src/scores.cpp

```diff
@@ -2210,8 +2279,30 @@ int loadGame(int player, int saveIndex)
 					fp->seek(sizeof(Sint32), File::SeekMode::ADD);
 				}
 				fp->seek(sizeof(Uint32) * 10, File::SeekMode::ADD); // equipment slots
+		    } else {
+			    // server needs to skip past other players' equipment
+			    // (this is stored differently)
+			    int i;
+			    for ( i = 0; i < 10; i++ )
+			    {
+				    int itemtype = NUMITEMS;
+				    fp->read(&itemtype, sizeof(ItemType), 1);
+				    if ( itemtype < NUMITEMS )
+				    {
+					    fp->seek(sizeof(Status), File::SeekMode::ADD);
+					    fp->seek(sizeof(Sint16), File::SeekMode::ADD);
+					    fp->seek(sizeof(Sint16), File::SeekMode::ADD);
+					    fp->seek(sizeof(Uint32), File::SeekMode::ADD);
+					    fp->seek(sizeof(bool), File::SeekMode::ADD);
+				    }
+			    }
 			}
 		}
+		else if ( multiplayer == CLIENT )
+		{
+			// client needs only to skip the dummy byte
+			fp->seek(sizeof(Status), File::SeekMode::ADD);
```

Comment:
Where does this dummy byte come from? cant recall

Reply:
I believe this should be `sizeof(ItemType)` since it writes as a dummy: `c = NUMITEMS;
					fp->write(&c, sizeof(ItemType), 1);`